resources/scripts/jquery/jquery-utils.js

```diff
@@ -308,6 +308,25 @@
     	$(".eXist_wizard_next", container).click(function () {
     		if (currentPage == pages.length - 1)
     			return;
+		var name = document.getElementsByName('name')[0].value;
+           var abbrev = document.getElementsByName('abbrev')[0].value;
+           var title = document.getElementsByName('title')[0].value;
+
+                try {
+                  nameURL = new URL(name);
```

Comment:
Does `name` has to be a valid URL?
I understood this is only a convention and any unique identifier is allowed here. Might have to be a valid `xs:id`.